src/Microsoft.ML.Transforms/CategoricalCatalog.cs

```diff
@@ -13,12 +13,17 @@ namespace Microsoft.ML
     public static class CategoricalCatalog
     {
         /// <summary>
-        /// Convert a text column into one-hot encoded vector.
+        /// Convert several text column into one-hot encoded vectors.
```

Comment:
>column [](start = 33, length = 6)

column"s"

Reply:
Thank you!

---
In reply to: [266970355](https://github.com/dotnet/machinelearning/pull/2959#discussion_r266970355) [](ancestors = 266970355)